GitAutoFetch_Dev16/source.extension.vsixmanifest

```diff
@@ -1,31 +1,31 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-	<Metadata>
-		<Identity Id="ca8e6794-e60f-460f-9abd-c10f926deb99" Version="1.1.6" Language="en-US" Publisher="Zwei Developer" />
-		<DisplayName>GitAutoFetch VS2022</DisplayName>
-		<Description xml:space="preserve">Auto fetch in Git, menu location in solution explorer (Git AutoFetch and Configure Fetching)</Description>
-		<Icon>Logo.png</Icon>
-		<Tags>git, autofetch, git-autofetch, fetch</Tags>
-	</Metadata>
-	<Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,17.0)">
-      <ProductArchitecture>x86</ProductArchitecture>
-    </InstallationTarget>
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.0,17.0)">
-      <ProductArchitecture>x86</ProductArchitecture>
-    </InstallationTarget>
-    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.0,17.0)">
-      <ProductArchitecture>x86</ProductArchitecture>
-    </InstallationTarget>
-	</Installation>
-	<Dependencies>
-		<Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-		<Dependency Id="Microsoft.VisualStudio.MPF.16.0" DisplayName="Visual Studio MPF 16.0" d:Source="Installed" Version="[16.0,)" />
-	</Dependencies>
-	<Prerequisites>
-		<Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
-	</Prerequisites>
-	<Assets>
-		<Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
-	</Assets>
+    <Metadata>
+        <Identity Id="ca8e6794-e60f-460f-9abd-c10f926deb99" Version="1.1.6" Language="en-US" Publisher="Zwei Developer" />
+        <DisplayName>GitAutoFetch</DisplayName>
+        <Description xml:space="preserve">Auto fetch in Git, menu location in solution explorer (Git AutoFetch and Configure Fetching)</Description>
+        <Icon>Logo.png</Icon>
+        <Tags>git, autofetch, git-autofetch, fetch</Tags>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,17.0)">
+            <ProductArchitecture>x86</ProductArchitecture>
```

Comment:
Will commit suggestions, but [ProductArchitecture metadata is not valid in v16.x of build tools.](https://developercommunity.visualstudio.com/t/schema-validation-error-after-using-vsix-manifest/1521924)